Solving #4073 New PR (#4272)
MicrosoftDocs/windows-itpro-docs: windows/client-management/mdm/policy-csp-authentication.md

This policy is intended for use on Shared PCs to enable a quick first sign-in experience for a user. It works by automatically connecting new non-admin Azure Active Directory (Azure AD) accounts to the pre-configured candidate local accounts.

> [!Important]

> > Pre-configured candidate local accounts: Means any **local account** (preconfigured or added) that you could have in the device.

Reviewer: ```suggestion
> Pre-configured candidate local accounts refers to any **local account** (pre-configured or added) in your device.
```
